src/entities/slop/ui/level-chip.tsx

```diff
@@ -30,18 +30,20 @@ const LEVEL: Record<Level, { text: string; color: string }> = {
 };
 
 interface LevelChipProps {
+  className?: string;
   level: Level;
 }
 
-const LevelChip = ({ level }: LevelChipProps) => {
+const LevelChip = ({ level, className }: LevelChipProps) => {
   return (
     <div
       className={cn(
         'flex h-[25px] w-[56px] items-center justify-center rounded-[6px] border-[1px] border-white border-opacity-10',
-        LEVEL[level].color
+        LEVEL[level].color,
+        className && className
```

Comment:
```suggestion
        className
```

그냥 className을 넣는건 어떨까!?